virttest/libvirt_xml/vm_xml.py

```diff
@@ -1570,6 +1570,29 @@ def remove_all_disk(self):
         """
         self.remove_all_device_by_type('disk')
 
+    @staticmethod
+    def set_vm_features(vm_name, **attrs):
+        """
+        Set attrs of vm features xml
+
+        :param vm_name: The name of vm to be set
+        :param attrs: attributes to be set
+        """
+        vmxml = VMXML.new_from_inactive_dumpxml(vm_name)
+        if vmxml.xmltreefile.find('/features'):
+            features_xml = vmxml.features
+        else:
+            features_xml = VMFeaturesXML()
+        try:
+            for attr_key, value in attrs.items():
```

Comment:
Should we use list(attrs.items()) here?

Reply:
I was thinking of that too. But I have tested this piece of code on python3, it works fine. So I guess it doesn't matter...?

Reply:
it return dict_items so I think it is needed to do transfer

Reply:
It is better to have because there is still python 2 test.
With python 3:
>>> a = {'a': 1}
>>> a.items()
dict_items([('a', 1)])

With python 2:
>>> a = {'a': 1}
>>> a.items()
[('a', 1)]

Reply:
Actually when it comes to a loop like 'for', this works fine with both python2 and python3

Reply:
Agreed. Here we only need an iterable and attrs.items() can work same with python 3 and python 2. So I approve .